src/Autofac.Integration.SignalR/Autofac.Integration.SignalR.csproj

```diff
@@ -1,87 +1,37 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{CCC12CB7-6753-4772-9B04-83A0B2554991}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Autofac.Integration.SignalR</RootNamespace>
-    <AssemblyName>Autofac.Integration.SignalR</AssemblyName>
-    <FileAlignment>512</FileAlignment>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\Autofac.snk</AssemblyOriginatorKeyFile>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <TargetFramework>net461</TargetFramework>
```

Comment:
Let's raise this to net472 since we dropped net461 from core Autofac and net461 has some odd near-incompatibility things with netstandard2. I'm adding a note to update all the .NET desktop framework things to be net472.

Reply:
If we introduce the changes to update the framework, should we also update some NuGet-Packages referenced?

Reply:
Sounds good. On it now. Somehow some Update of Windows causing problem with IDE. Hoping to get it done soon 🤷‍♂️

Reply:
@tillig just noticed that `2.4.1`  of `Microsoft.AspNet.SignalR.Core` has a dependency to `Microsoft.Owin.Security` >= `2.1.0`. Currently we target `3.0.0`. I think that would be another breaking change, even though the tests are still passing, which might be a false positive. 

Checked in with dd9eed0a0dec1dd4fab745b250539b0c6ee40536 for now. Please let me know what you think.

Reply:
Uhhhh let's just leave it. I mean, it's a 6.0 full version update anyway, so if it's a break, it's a break. Semver, baby! 😆 I think _down-versioning_ is probably OK, I mean, technically, our assembly doesn't really care anyway.